tabulator/parsers/json.py

```diff
@@ -53,7 +53,7 @@ def __emit_items(self):
             prefix = '%s.item' % self.__path
         items = ijson.items(self.__chars, prefix)
         for item in items:
-            if isinstance(item, list):
+            if isinstance(item, (tuple, list)):
```

Comment:
@pwalsh 
Sorry I haven't answered it yet but I've checked `types.GeneratorType` it doesn't work for e.g. `iter([1,2,3]). So I've decided to pass to `NativeLoader/Parser` all data types except file-like and strings - it's in `detect_scheme/detect_format`.

But check here is about rows:
```
[
    ('a','b','c'), # tuple
    (1,2,3), # tuple
    [1,2,3], # list
]
```
So almost there is no chance to have here generators/iterators and set is not ordered. It looks typle-list could be enough. WDYT?